Renaming of the Developer Portal (#7804)
mendix/docs: content/en/docs/developerportal/_index.md

---
title: "Apps"
url: /developerportal/
description: "Describes the home page of Apps and links to more detailed documents in the guide."
weight: 30
no_list: false
description_list: true
cascade:
    - content_type: "Mendix Portal for Developers"
    - mendix_version: 10
#If moving or renaming this doc file, implement a temporary redirect and let the respective team know they should update the URL in the product. See Mapping to Products for more details.
#This document is mapped to the landing page, update the link there if renaming or moving the doc file.
---

{{% alert color="info" %}}

> The Mendix Portal is the whole online platform of Mendix, which includes Apps, [Control Center](/control-center/), [Community](/community-tools/), [Marketplace](/appstore/), [Catalog](/catalog/), and [Mendix Support](/support/).

Reviewer: Looks pretty good. I don't really like the word "whole" there. I wonder if a minor change would improve things.
Perhaps we can discuss it at the group meeting and see if anyone has any further improvements - I think it is quite important to get it right.
```suggestion
The Mendix Portal is the online platform of Mendix. It includes Apps, [Control Center](/control-center/), [Community](/community-tools/), [Marketplace](/appstore/), [Catalog](/catalog/), and [Mendix Support](/support/).
```
